.eslintrc.js

```diff
@@ -17,7 +17,22 @@ module.exports = {
         "eslint-plugin/prefer-placeholders": "error",
         "eslint-plugin/report-message-format": ["error", "[^a-z].*\\.$"],
         "eslint-plugin/require-meta-type": "error",
-        "eslint-plugin/test-case-property-ordering": "error",
+        "eslint-plugin/test-case-property-ordering": [
+            "error",
+
+            // https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/79
+            [
+                "filename",
+                "code",
+                "output",
+                "options",
+                "parser",
+                "parserOptions",
+                "globals",
+                "env",
+                "errors"
+            ]
+        ],
```

Comment:
Can we make this change in a separate PR? (To avoid merge conflicts later, we could merge it as a `Chore` soon, before v6 prereleases start.)

Reply:
Done #11511